configman/config_manager.py

```diff
@@ -461,7 +461,10 @@ def write_conf(self, config_file_type, opener, skip_keys=None):
                 an_option = option_defs[a_key]
                 if ((not a_key.startswith('admin'))
                     and isinstance(an_option, Option)
-                    and an_option.secret
+                    and (
+                        an_option.secret
+                        or 'password' in an_option.name.lower()
+                    )
```

Comment:
Sorry. I blame it being post-vacation but I don't see how that connects. By default we **don't** expose secrets. So, by default this code is run and all things called "password" something gets redacted. But amongst that what if you have an example like `password_hint` or `password_minimum_length` or something. Then you want the ini file to say:

```ini
[stuff]
#password=****************

#password_hint="Mother's maiden name?"

#min_password_length=6
```